src/docs/asciidoc/tcp-client.adoc

```diff
@@ -358,6 +362,37 @@ public class Application {
 ----
 ====
 
+If you need to specify an idle time for the channels in the connection pool, you can apply the following configuration:
+
+====
+[source,java]
+----
+import reactor.netty.Connection;
+import reactor.netty.resources.ConnectionProvider;
+import reactor.netty.tcp.TcpClient;
+import java.time.Duration;
+
+public class Application {
+
+    public static void main(String[] args) {
+        ConnectionProvider provider = ConnectionProvider.fixed("fixed", 50, 30000, Duration.ofSeconds(60));
+        Connection connection =
+                TcpClient.create(provider)
+                         .host("example.com")
+                         .port(80)
+                         .connectNow();
+
+        connection.onDispose()
+                  .block();
+    }
+}
+----
+====
+
+NOTE: Be cautious with "`elastic`" connection pool when you expect a high load. You might experience
```

Comment:
Add "an" after "with" on this line, please.

Reply:
done